usr/src/boot/common/module.c

```diff
@@ -80,8 +80,12 @@ static vm_offset_t loadaddr = 0;
 #if defined(LOADER_FDT_SUPPORT)
```

Comment:
I'd use #elif again.  We should probably do something more correct with `LOADER_FDT_SUPPORT` too, long term.

Reply:
Yeah, this is a leftover from FreeBSD, and not actually something we use (or really need).